File: app/src/main/java/com/kruma/kanta/service/IProductoTipo.java
package com.kruma.kanta.service;

import com.kruma.core.service.entity.ServiceCallback;
import com.kruma.core.util.common.List;
import com.kruma.kanta.entidad.ProductoTipo;

import retrofit.http.GET;

/**
 * Created by Carlos Gomez on 28/10/2016.
 */

public interface IProductoTipo {
    @GET("/Listar")
    void Listar(ServiceCallback<List<ProductoTipo>> obj_pCallBack);
}

File: KantaPe_Web/Kruma.KantaPe/Kruma.KantaPe.Service/App_Code/Interface/IProductoTipo.cs
using System.ServiceModel;
using System.ServiceModel.Web;

namespace Kruma.KantaPe.Service.Interface
{
    [ServiceContract]
    public interface IProductoTipo
    {
        [OperationContract]
        [WebInvoke(
            Method = "GET",
            ResponseFormat = WebMessageFormat.Json,
            BodyStyle = WebMessageBodyStyle.Bare,
            UriTemplate = "Listar")]
        Kruma.KantaPe.Service.Entity.ServiceResult<
            Kruma.Core.Util.Common.List<
                Kruma.KantaPe.Entidad.ProductoTipo>> Listar();
    }
}